api/run_eval.py

```
@@ -257,9 +262,117 @@ def transcribe_with_retry(
+
+                result_url = post_response.get("result_url")
+                if result_url:
+                    while True:
+                        poll_response = make_request(result_url, headers)
+                        if poll_response.get("status") == "done":
+                            result = poll_response.get("result")
+                            break
+                        elif poll_response.get("status") == "error":
+                            raise Exception(
+                                f"Gladia transcription failed: {poll_response.get('error', 'Unknown error')}"
+                            )
```

Ruff (0.13.2)

Abstract `raise` to an inner function

(TRY301)